service/src/main/java/uk/gov/hmcts/reform/fpl/model/Respondent.java

```diff
@@ -86,6 +94,24 @@ public boolean containsConfidentialDetails() {
         return hiddenValue.equals("Yes");
     }
 
+    @JsonIgnore
+    public boolean hasRegisteredOrganisation() {
+        return Optional.ofNullable(getSolicitor()).flatMap(
+            respondentSolicitor -> Optional.ofNullable(respondentSolicitor.getOrganisation()).map(
+                organisation -> isNotBlank(organisation.getOrganisationID())
+            )
+        ).orElse(false);
+    }
+
+    @JsonIgnore
+    public boolean hasUnregisteredOrganisation() {
+        return Optional.ofNullable(getSolicitor()).flatMap(
+            respondentSolicitor -> Optional.ofNullable(respondentSolicitor.getUnregisteredOrganisation()).map(
+                organisation -> isNotBlank(organisation.getName())
```

Comment:
may need to validate address is present as well (should address be mandatory? or just name? or both?) check with business